ras_party/controllers/queries.py

```diff
@@ -52,6 +52,18 @@ def query_respondent_by_email(email, session):
     return session.query(Respondent).filter(func.lower(Respondent.email_address) == email.lower()).first()
 
 
+def query_respondent_by_email_filter_out_created(email, session):
+    """
+    Query to return respondent based on email
+    :param email: the party uuid
+    :return: respondent or none
+    """
+    logger.debug('Querying respondents by email', email=email)
```

Comment:
Are we ok to log out email addresses?

Reply:
Damn thought I caught all them. Good catch